target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/vr9_avm_fritz7360-v2.dts

```diff
@@ -4,7 +4,6 @@
 #include "vr9_avm_fritz736x.dtsi"
 
 #include <dt-bindings/input/input.h>
-#include <dt-bindings/mips/lantiq_rcu_gphy.h>
```

Comment:
Same is true for input.h as well, so you may remove both at the same time.

Reply:
Both lines are removed now.